arch/arm/dts/crocodile-u-boot.dtsi

```diff
@@ -68,6 +68,11 @@
 	u-boot,dm-spl;
 };
 
+&uart4 {
+	// marker for comon/leica_sep.c to find the device-node by
+	leica-sep-node;
```

Comment:
SEP is a protocol - it should be a marked as PP4 or EFI node

Reply:
ACK, i'll change it to - "leica-MCU-uart"? (even though MCU is... generic; but at least better than the misnomed/confusing "extensible firmware interface"; or the more cryptic PP4 - both leica specific/internal abbreviations)